haunt-roll-fail/arcs/game.scala

```diff
@@ -1480,7 +1480,7 @@ class Game(val setup : $[Faction], val options : $[Meta.O]) extends BaseGame wit
                 XXSelectObjectsAction(f, f.hand)
                     .withGroup(Farseers, "discard cards to redraw")
                     .withThen(l => FarseersRedrawAction(f, l, then))(l => ("Discard to draw", (l.num + 1).cards))
-                    .withExtras(NoHand, CancelAction)
+                    .withExtras(NoHand, FarseersRedrawAction(f, $, then).as("Skip"), CancelAction)

```

Comment:
"Skip" would be misleading here, `as("Discard", Farseers, "to draw", 1.cards)` might be more clear

Reply:
Texts updated.